notification_configuration.go

```diff
@@ -96,7 +96,7 @@ type DeliveryResponse struct {
 	Body       string      `json:"body"`
 	Code       int         `json:"code"`
 	Headers    http.Header `json:"headers"`
-	SentAt     time.Time   `json:"sent-at,iso8601"`
+	SentAt     time.Time   `json:"sent-at"`
```

Comment:
? I don't understand this one; is this a no-op?

Reply:
it's a no-op, just a lint fix:

```
notification_configuration.go:99:25: SA5008: unknown JSON option "iso8601" (staticcheck)
	SentAt     time.Time   `json:"sent-at,iso8601"`
	                       ^
```

Reply:
fine dropping the commit, but it's just nonsense the json parser doesn't understand

Reply:
I think this should be fine. I'm not seeing any custom [changes in the forked jsonapi](https://github.com/svanharmelen/jsonapi/blame/master/request.go#L505-L520).

@mwhooker I [just merged](https://github.com/hashicorp/go-tfe/pull/195) a change to extract the unmarshalling bit into a separate function so it makes it easier to validate these parsing changes.

Here is an example test that it uses to [build a response body](https://github.com/hashicorp/go-tfe/blob/master/tfe_test.go#L408-L431) and it calls [unmarshalResponse](https://github.com/hashicorp/go-tfe/blob/master/tfe_test.go#L437).

Is it possible to do that here? Feel free to message me on Slack

Reply:
Hey @omarismail, Will see if I can take a look at this another day. I probably don't have too much more time to work on this PR. I'll just revert the commit if there's follow-on work you think should be done.